app/assets/stylesheets/app/font.scss

```diff
@@ -0,0 +1,25 @@
+/* define fonts */
+@font-face {
```

Comment:
Can you provide a link to where the font comes from? (I believe https://rsms.me/inter/). I had a fear that maybe it had some sort of commercial license. Looks like we're all good there, though.

Reply:
@robstolarz might have to answer this one.